rfcs/text/0001-rfc-process.md

```diff
@@ -103,6 +161,22 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
   
      ***@channel**: Call for community review of the RFC process - https://github.com/HumanCellAtlas/dcp-community/pull/27 - Last call is April 1.*
 
+**ADDITION BEGINS**
+
+- For software RFCs, it is recommended that an FYI be sent to the HumanCellAtlas **#tech-architecture** slack channel. Include a link to the RFC pull request and the last call deadline.
+
+  **Recommended:** Add the RFC as an agenda item to the next *DCP Architecture* meeting for spirited discussion
+
+- For governance RFCs, it is recommended that an FYI be sent to the HumanCellAtlas **#dcp-project-mgmt** slack channel. Include a link to the RFC pull request and the last call deadline.
+    
+  **Recommended:** Add the RFC as an agenda item to the next *DCP PM* meeting for spirited discussion
+
+**Example Slack Channel Message** 
```

Comment:
Editorial: Match the style of previous example messages.

Reply:
Done. I think this went away entirely during reformatting.